code/Model/Submission/SubmittedFormField.php

```diff
@@ -19,7 +19,8 @@ class SubmittedFormField extends DataObject
     private static $db = [
         'Name' => 'Varchar',
         'Value' => 'Text',
-        'Title' => 'Varchar(255)'
+        'Title' => 'Varchar(255)',
+        'IsVisible' => 'Boolean',
```

Comment:
This is not needed, right? The field will carry it from the controller without being backed up by a true db field.

Reply:
That's correct. I was on the fence here as well. But in the end I thought it might be beneficial to have that information available later on, maybe for an data export or similar?

Reply:
Since it can be always calculated from the data, I'm for not having it.

Reply:
hm.. the visibility could change over time, plus you would have to re-create the submitted form `$data` array to use `EditableFormField::isDisplayed()` to recalculate it. I think storing the flag is the better option? But if this is what it takes to get this accepted, I'll remove it.

Reply:
If the visibility state of the form changes after the form was submitted, the newly calculated value will not be correct for the submission. I'm in favor of it being stored too.

Reply:
Ok, fair enough. Can we consider naming it `Displayed` then since it reflects a past state?